util/util.go

```diff
@@ -1,9 +1,37 @@
 package util
 
-import "os/exec"
+import (
+	"errors"
+	"net"
+	"os/exec"
+	"strings"
+)
 
 // IsCommandAvailable check command exits.
 func IsCommandAvailable(cmd string) bool {
 	_, err := exec.LookPath(cmd)
 	return err == nil
 }
+
+func GetClientIp() (string, error) {
+	addresses, err := net.InterfaceAddrs()
+
+	if err != nil {
+		return "", err
+	}
+
+	for _, address := range addresses {
+		if ipnet, ok := address.(*net.IPNet); ok && !ipnet.IP.IsLoopback() {
+			if ipnet.IP.To4() != nil {
+				return ipnet.IP.String(), nil
+			}
+
+		}
+	}
+
+	return "", errors.New("can not find the client ip addresses")
+}
+
+func CountWords(s string) int {
+	return len(strings.Fields(s))
+}
```

Comment:
Overall, the code patch appears to be well-written and follows best practices. However, there are a few suggestions for improvement:

1. Error handling: The IsCommandAvailable function only returns a boolean indicating whether the command is available or not, but it does not provide any error information. It might be useful to return an error message that provides more context about why the command is not available.

2. Input validation: The GetClientIp function assumes that the net.InterfaceAddrs function will always return a valid list of addresses. It would be a good idea to add some input validation to ensure that the function does not break if there are no addresses available.

3. Security: The GetClientIp function retrieves the IP address of the client, but it does not perform any authentication or authorization checks. If this function is used in a security-sensitive context, it might be necessary to add some additional checks to ensure that the client is authorized to access the system.

4. Code redundancy: The patch imports the "os/exec" package twice, which is unnecessary. It would be better to remove the duplicate import statement.

5. Function naming: The CountWords function is a bit generic and could be more specific. Perhaps a better name would be CountWordsInString or CountWordsInText.